function.rb

```diff
@@ -0,0 +1,9 @@
+
+def function(n)
+  if n == 0
+    1
+  else
+    n * function(n-1)
+  end
+end
```

Comment:
What is being returned?

Reply:
Just to add , by default, functions return undefined. To return any other value, the function must have a return statement that specifies the value to return.